charts/kthena/charts/workload/templates/kthena-controller-manager/component/mutating-webhook.yaml

```diff
@@ -20,22 +20,22 @@ webhooks:
       service:
         name: {{ include "kthena.name" . }}-webhook
         namespace: {{ .Release.Namespace }}
-        path: "/mutate-registry-kthena-ai-v1alpha1-model"
+        path: "/mutate-workload-serving-volcano-sh-v1alpha1-modelbooster"
```

Comment:
I would suggest we donot maintain so long path, prefer the short ones like /modelbooster, /modelserving

It is more simpler to maintain even if we rename or upgrade to v1beta1